demos/common/stream_client/stream_client.py

```diff
@@ -172,7 +173,7 @@ def display(self):
                 self.pq.put((sent_frame_id, received_frame, timestamp))
 
     def get_timestamp(self) -> int:
-        return int(cv2.getTickCount() / cv2.getTickFrequency() * 1e6)
+        return int(time.time() * 1e6)
```

Comment:
why this change? why do we still want to stamp it manually? should we rely on automatic stamping?
otherwise the metric is wrong - we take into account client->server network transfer time

Reply:
fixed that, now benchmarking is based on server side timestamps. Client side timestamps are now only used to order frames while using legacy(non-streaming) method.